src/Constants.h

```diff
@@ -69,6 +69,10 @@
 // Shared region polygon approximation
 #define DEFAULT_VERTEX_COUNT 1000
 
+// file list
+#define REPORT_FIRST_PROGRESS_AFTER_SECS 5
```

Comment:
what files are these used in? As specified in #761 , I'd like to refactor this file

Reply:
to avoid rebuilds

Reply:
They are used in files `src/FileList/FileListHandler.cc` and `src/Table/TableController.cc`.